### ISSUE TITLE
Their is no W365 category on the roadmap site. 

### ISSUE THREAD

**Issue by Rkendall75** [MicrosoftDocs/memdocs#2080]
The article says to reference the Microsoft roadmap for what's new (in addition to the website) but their is no entries or categories for W365. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 6ab7e8fe-9bbb-9ad3-4fcf-a8b5ef6fcfaa
* Version Independent ID: ca7ef4d8-ba05-9698-1e3c-430eaebb102a
* Content: [In development - Windows 365 Enterprise](https://docs.microsoft.com/en-us/windows-365/enterprise/in-development)
* Content Source: [windows-365/enterprise/in-development.md](https://github.com/MicrosoftDocs/memdocs/blob/master/windows-365/enterprise/in-development.md)
* Service: **cloudpc**
* Sub-service: ****
* GitHub Login: @ErikjeMS
* Microsoft Alias: **erikje**

**Comment by ErikjeMS** [comment]
@Rkendall75 You're right! I'll remove that bullet point until the Roadmap includes Win365.
